sisyphus/simple_linux_utility_for_resource_management_engine.py

```diff
@@ -2,6 +2,7 @@
 
 from collections import defaultdict, namedtuple
 from enum import Enum
+from typing import Callable
```

Comment:
This import isn't needed if Callable is only mentioning in the comments and not used as type hint.

Reply:
```suggestion
```